docs/server/source/production-deployment-template/template-kubernetes-azure.rst

```diff
@@ -128,14 +128,14 @@ You can SSH to one of the just-deployed Kubernetes "master" nodes
 
 .. code:: bash
 
-   $ ssh -i ~/.ssh/<name>.pub ubuntu@<master-ip-address-or-hostname>
+   $ ssh -i ~/.ssh/<name> ubuntu@<master-ip-address-or-hostname>
```

Comment:
@muawiakh and I discussed this. If we specify the `.pub` while creating the cluster, then we use the private key to access it. If we use the private key while creating the cluster, then we use the public key with `.pub` extension.

And since a private key should never leave the local system, we should specify the `.pub` key in the command to create the cluster, and use the private key to log in to the cluster.